clean.go

```diff
@@ -113,3 +110,39 @@ func clean(pkgs []string) {
 		}
 	}
 }
+
+func keepFile(filename string) bool {
+	if strings.HasSuffix(filename, "_test.go") {
+		return false
+	}
+	if strings.HasSuffix(filename, ".go") {
+		return true
+	}
+	if strings.HasSuffix(filename, ".s") {
+		return true
+	}
+	if isLegalFile(filename) {
+		return true
+	}
+	return false
+}
+
+var commonLegalFilePrefixes = []string{
+	"license",
```

Comment:
I would consider copying the list from https://github.com/tools/godep/blob/master/license.go

Sadly that whole file has exported methods but they are in package main or I would just depend on and use that.